main.py

```diff
@@ -1,7 +1,45 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Actor, Genre
+
+genres = ["Western", "Action", "Dramma"]
+actors = [("George", "Klooney"),
+          ("Kianu", "Reaves"),
+          ("Scarlett", "Keegan"),
```

Comment:
There is a potential issue with having two actors with the first name 'Scarlett'. This might lead to unintended deletions later in the code.